environment/templates/chainlink-deployment.yml

```diff
@@ -9,7 +9,7 @@ spec:
           secretName: {{ .Manifest.Secret.ObjectMeta.Name }}
       containers:
       - name: node
-        image: {{ .Config.Apps.Chainlink.Image }}:{{ .Config.Apps.Chainlink.Version }}
+        image: {{ if .Values.chainlinkCluster.chainlink.images }}{{ iterateImage .Values.chainlinkCluster.chainlink.images }}{{ else }}{{ .Config.Apps.Chainlink.Image }}{{ end }}:{{ if .Values.chainlinkCluster.chainlink.versions }}{{ iterateVersion .Values.chainlinkCluster.chainlink.versions }}{{ else }}{{ .Config.Apps.Chainlink.Version }}{{ end }}
```

Comment:
Logic being added to deployment files should be avoided if possible since it can be harder to follow the logic when you have to jump out of the code. Would it be easier to update the config values used before the deployment file is read and used?